src/service/sync.ts

```diff
@@ -41,7 +41,7 @@ async function setup(trx: Knex.Transaction, context: SynchronizeContext): Promis
     log('PRE-SYNC: End');
   }
 
-  // Run the migration scripts.
+  // Run the synchronization scripts.
```

Comment:
```suggestion
  // Run the sync scripts for views, functions, and xwalks.
```

Reply:
Don't think we need to all object names in comment. sync-db support triggers sync as well recently. so a generic comment should be enough IMO ` Run the synchronization scripts.`